docs/kb/api-version-3/custom-redirects.md

```diff
@@ -75,6 +75,8 @@ For a complete list of fields involved in user operations, see the [Custom Redir
 | **Operation** | **URL Format**                             | **Required Parameters** | **Description**  |
 | ------------- | ------------------------------------------ | ----------------------- | -----------------|
 | `read` | `/api/customRedirect/version/3/do/read/id/`**_`<id>`_**`?...` | `user_key, api_key, id` | Returns the data for the custom redirect specified by `<id>`. `<id>` is the Pardot ID of the target custom redirect. |
+| `create` | `/api/customRedirect/version/3/do/create?... | `user_key, api_key, id` | Creates a new custom redirect using the specified data. |
```

Comment:
missing an apostrophe after: `/api/customRedirect/version/3/do/create?...

Reply:
thank you